pkg/locale/locale.go

```diff
@@ -36,3 +42,15 @@ func NewLocale(l string) Locale {
 func (l *locale) GetLocale() string {
 	return l.locale
 }
+
+func getAcceptLang(ctx context.Context) string {
+	md, ok := metadata.FromIncomingContext(ctx)
+	if !ok {
+		return JaJP
```

Comment:
We can't use it because we don't write most fields in en.yaml. Thus, it occurs errors.

Reply:
It's true. We spoke about this.
So we can't merge the following PR until we translate all fields. If not, it will be an error, as you said.
https://github.com/bucketeer-io/bucketeer/pull/213 

Can you cherry-pick the commit from the above PR and deploy it tomorrow in the dev environment to ensure it is working?

Reply:
Ok, I'll let you know the result.